testsuite/tests/apicast/operator/conftest.py

```diff
@@ -0,0 +1,181 @@
+"provide fixtures for custom policy through operator testing"
+import pytest
+import backoff
+
+from testsuite import rawobj
+from testsuite.utils import blame
+from testsuite.openshift.objects import Secrets
+
+
+@pytest.fixture(scope="module")
+def secrets(openshift):
+    """Interface to return the dict-like Secrets
+    """
+    return Secrets(openshift())
+
+
+@pytest.fixture(scope="module")
+def delete_secret(secrets):
+    """Delete the secret containing the policy after execution
+    """
+    yield
+    del secrets["policy"]
+
+
+# pylint: disable=unused-argument
+@pytest.fixture(scope="module")
+def create_custom_policy_secret(secrets, delete_secret):
+    """Create an openshift secrets to use as custom policy based on https://github.com/3scale-qe/apicast-example-policy
+    """
+    initlua = "return require('example')"
+    apicastpolicyjson = """
+    {
```

Comment:
I would put the policy into separate resource files as we would probably like to reuse it as much as possible.

Reply:
I can move it to the root contest if you and @mganisin agree.

Reply:
Not root conftest but rather separate files which you can then easily load in the code

Reply:
of course i was thinking about the same but i didnt comment at the end as i have no strong opinion. So far I had hard times to find a case where to reuse it.

Reply:
There will be a separate testcase for APIcast operator, which doesn't have to be disruptive and be run more often.

I was also thinking about rewriting some of our custom policy tests so that they do not need a separate repository with code (might be not possible)